.docker/docker-compose.local.yml

```diff
@@ -38,13 +39,35 @@ services:
     env_file:
       - ../.env/api.env
     networks:
+      - socket_network
       - backend_network
       - frontend_network
     logging:
       driver: "json-file"
       options:
         max-size: "10m"
         max-file: "5"
+  socket:
```

Comment:
Use the same name for your server, if you prefer a socket server name use it in all places...